docs/handler.md

```diff
@@ -0,0 +1,63 @@
+# `gpii.express.handler`
+
+An abstract grade for "request handler" modules.  Modules that extend this grade are expected to be created
+dynamically, as outlined in [the Fluid documentation](http://docs.fluidproject.org/infusion/development/SubcomponentDeclaration.html#dynamic-subcomponents-with-a-source-event).
+
+These are used with a `gpii.express.middleware` module such as the `requestAware` and `contentAware` grades included in
+this package.  The middleware is expected to construct one of these components per request.  Note that these
+components are not persisted.  Any data you wish to retain should be stored elsewhere, for example, by relaying it
+to the parent middleware.
+
+The simplest implementation uses the built-in `sendResponse` invoker (see below), as in the following example:
+
+```
+invokers: {
+  handleRequest: {
+    func: "{that}.sendResponse",
+    args: [ 200, "I am happy to hear from you whatever you have to say." ]
+  }
+}
+```
+
+For more examples of how this can be used, check out the tests included with this package.
+
+
+# Component Options
+
+| Option     | Type       | Description |
+| ---------- | ---------- | ----------- |
+| `request`  | `{Object}` | The [request object](http://expressjs.com/en/api.html#req) provided by Express, which wraps node's [`http.incomingMessage`](https://nodejs.org/api/http.html#http_class_http_incomingmessage). The `requestAware` middleware grade and its derivatives (such as the `contentAware` middleware) set this for you. |
```

Comment:
See above - these bits of duplicate text should just link to a single source where they are defined once and for all